tests/conftest.py

```diff
@@ -115,6 +117,10 @@ def ssh_client_session(ssh_session_connect):
     # noqa: DAR101
     """
     ssh_session = Session()
+
+    if os.getenv('LIBSSH_VERBOSITY'):
+        ssh_session.set_log_level(logging.DEBUG)
```

Comment:
Let's move this to a separate PR, as it's not related to the segfault. It'll need to come in through a fixture interface and avoid relying on env vars. That fixture will need to read pytest's own verbosity level instead of having an additional toggle for the same thing, similar to this example: https://docs.pytest.org/en/stable/reference/reference.html#pytest.fixtures.pytestconfig.

Reply:
Removed. Thanks for suggestion. I will look into this in separate PR.